src/js/intl-tel-input.ts

```diff
@@ -2151,10 +2151,7 @@ const loadUtils = (source: string|UtilsLoader): Promise<unknown> | null => {
       loadCall = import(/* webpackIgnore: true */ /* @vite-ignore */ source);
     } else if (typeof source === "function") {
       try {
-        loadCall = source();
-        if (!(loadCall instanceof Promise)) {
-          throw new TypeError(`The function passed to loadUtils must return a promise for the utilities module, not ${typeof loadCall}`);
-        }
+        loadCall = Promise.resolve(source());
```

Comment:
@jackocnr This seems fine to me. 👍 The main thing worth considering is that this will also allow the `loadUtilsOnInit` function to return the utils module object directly instead of just as a promise. That is, you can now do:

```js
intlTelInput(inputElement, {
  // No promises involved!
  loadUtilsOnInit: () => ({
    default: { /* Locally defined utils module, or a mock, or whatever here */ }
  })
});
```

But that flexibility might be useful! So I think this is probably a good change. (Side note: the function can technically return *anything* with this change, but we have code later [line 2168] to validate that it at least vaguely looks like a JS module object, so that should be OK.)

OTOH, if you want to keep requiring that this returns a *thennable* (the more generic form of a JS promise), then the right canonical check would be something like:

```js
loadCall = source();
if (typeof loadCall?.then !== 'function') {
  // Throw an error (or maybe return a rejection instead)
}
```

…but like I said, I think the added flexibility in this change as-is is probably better.